src/GridFS/StreamWrapper.php

```diff
@@ -57,6 +57,14 @@ class StreamWrapper
     /** @var ReadableStream|WritableStream|null */
     private $stream;
 
+    public function __destruct()
+    {
+        /* This destructor is a workaround for PHP trying to use the stream well
+         * after all objects have been destructed. This causes autoloading
+         * issues and possibly segmentation faults during PHP shutdown. */
```

Comment:
Nit: "This can causes", since apparently this isn't an issue for PHP 8 (if I understand Sara's response in the JIRA issue correctly).